app/controllers/contacts_controller.rb

```diff
@@ -4,7 +4,11 @@ class ContactsController < ApplicationController
 
   def index; end
 
-  def show; end
+  def show
+    @interaction = Interaction.new
+    @interaction.notes.build
+    @interactions = @contact.interactions
```

Comment:
Maybe use   @interactions = @contact.interactions`.includes(:notes)` here 👻 

If the app gets bigger and people start using it for their work (my sister for instance), they will have many (many!) contacts. This would preload interactions with their associated notes to prevent the N+1 query thingy